db/schema.rb

```diff
@@ -31,7 +40,7 @@
     t.uuid "uuid", default: -> { "uuid_generate_v4()" }
   end
 
-  create_table "case_worker_claims", force: :cascade do |t|
+  create_table "case_worker_claims", id: :serial, force: :cascade do |t|
```

Comment:
Did you deliberately add the `id: :serial` or have you had a recent pg update? 🤔

Reply:
not deliberate no. its from an update to activerecord i believe - happened on the last migration i did too.